src/Grok.Net/Grok.cs

```diff
@@ -10,6 +10,7 @@ namespace GrokNet
 {
     public class Grok
     {
+        private const string customPatternFileName = "grok-custom-patterns";
```

Comment:
Can we avoid of defining the file name and follow this: 
https://github.com/Marusyk/grok.net/issues/15#issuecomment-860511754 ?

Reply:
So we look for a directory named 'patterns' and load all the files present inside that?

Reply:
Correct

Reply:
Could you please remove this const? It seems we don't  neet it anymore